modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -154,7 +276,6 @@ spec:
         key: config
 ----
```

Comment:
The secret of [this line](https://github.com/mulesoft/docs-service-mesh/blame/v1.1/modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc#L238) contains user credentials instead of client ones. Is it possible to change those parameters with:
```
      "clientId": "_replace-with-connected-app-id_",
      "clientSecret": "_replace-with-connected-app-secret_"
```

Reply:
Done! Thanks!